pkg/asset/manifests/topologies_test.go

```diff
@@ -0,0 +1,143 @@
+package manifests
+
+import (
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/installer/pkg/types"
+	"k8s.io/utils/ptr"
+)
```

Comment:
Hmm, golint and gofmt tests are complaining for this file. Could you run a fmt/lint?
```bash
hack/go-fmt.sh .
hack/go-lint.sh
```

Reply:
Yeah odd, fixing now.